BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -264,7 +264,12 @@ extension AttributedLabel {
             if !isMeasuring {
                 if previousAttributedText != attributedText {
                     links = attributedLinks(in: model.attributedText) + detectedDataLinks(in: model.attributedText)
-                    accessibilityLinks = accessibilityLinks(for: links, in: model.attributedText)
+                    accessibilityLinks = accessibilityLinks(
+                        for: links,
+                        in: model.attributedText,
+                        accessibilityValue: environment.linkAccessibility
```

Comment:
it gets set as an `accessibilityValue` on the link element itself, with the link text being set as the `accessibilityLabel`, i wanted to avoid potential confusion around that.